
> Using Truffle library from global node_modules.

> server:            http://127.0.0.1:8555
> truffle:           v5.4.6
> ganache-core:      v2.13.0
> solidity-coverage: v0.7.17

Network Info
============
> id:      *
> port:    8555
> network: soliditycoverage


Instrumenting for coverage...
=============================

> YRX3.sol

Compiling your contracts...
===========================
> Compiling ./.coverage_contracts/YRX3.sol
> Compilation warnings encountered:

    project:/.coverage_contracts/YRX3.sol:298:9: Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
        string memory errorMessage
        ^------------------------^

> Artifacts written to /app/.coverage_artifacts/contracts
> Compiled successfully using:
   - solc: 0.5.16+commit.9c3226ce.Emscripten.clang


Compiling your contracts...
===========================
> Everything is up to date, there is nothing to compile.


[0m[0m
[0m  Contract: YRX3[0m
  [31m  1) test for YRX3[0m

    Events emitted during test:
    ---------------------------

    Ownable.OwnershipTransferred(
      previousOwner: <indexed> [33m0x0000000000000000000000000000000000000000[39m (type: address),
      newOwner: <indexed> [33m0x8D550643B994DFf9f743489E78Af5a7D9Be8781E[39m (type: address)
    )


    ---------------------------
  [31m  2) test for YRX3[0m

    Events emitted during test:
    ---------------------------

    Ownable.OwnershipTransferred(
      previousOwner: <indexed> [33m0x0000000000000000000000000000000000000000[39m (type: address),
      newOwner: <indexed> [33m0xeb39c78DE217152b9564799E69262f60ef4D1FC9[39m (type: address)
    )


    ---------------------------
  [32m  ‚úì[0m[90m test for YRX3[0m[31m (988ms)[0m
  [32m  ‚úì[0m[90m test for YRX3[0m[31m (243ms)[0m
  [31m  3) test for YRX3[0m
    > No events were emitted
  [32m  ‚úì[0m[90m test for YRX3[0m[31m (298ms)[0m
  [32m  ‚úì[0m[90m test for YRX3[0m[31m (255ms)[0m
  [32m  ‚úì[0m[90m test for YRX3[0m[31m (185ms)[0m
  [31m  4) test for YRX3[0m

    Events emitted during test:
    ---------------------------

    Ownable.OwnershipTransferred(
      previousOwner: <indexed> [33m0x0000000000000000000000000000000000000000[39m (type: address),
      newOwner: <indexed> [33m0xdB7129B7Bf7a30783e4ec0A42b7A86e0718dc630[39m (type: address)
    )


    ---------------------------
  [32m  ‚úì[0m[90m test for YRX3[0m[31m (269ms)[0m
  [32m  ‚úì[0m[90m test for YRX3[0m[31m (187ms)[0m
  [31m  5) test for YRX3[0m

    Events emitted during test:
    ---------------------------

    Ownable.OwnershipTransferred(
      previousOwner: <indexed> [33m0x0000000000000000000000000000000000000000[39m (type: address),
      newOwner: <indexed> [33m0x8D550643B994DFf9f743489E78Af5a7D9Be8781E[39m (type: address)
    )


    ---------------------------


[92m [0m[32m 7 passing[0m[90m (5s)[0m
[31m  5 failing[0m

[0m  1) Contract: YRX3
       test for YRX3:
[0m[31m     TypeError: Cannot read property 'call' of undefined[0m[90m
      at Context.<anonymous> (tests/test-YRX3.js:19:41)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
[0m
[0m  2) Contract: YRX3
       test for YRX3:
[0m[31m     AssertionError: expected promise to be rejected with 'Error' but it was fulfilled with {}[0m[90m
  
[0m
[0m  3) Contract: YRX3
       test for YRX3:
[0m[31m     Error: Returned error: sender account not recognized[0m[90m
      at Context.<anonymous> (tests/test-YRX3.js:56:36)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
[0m
[0m  4) Contract: YRX3
       test for YRX3:
[0m[31m     AssertionError: expected '0xdB7129B7Bf7a30783e4ec0A42b7A86e0718dc630' to equal 2.3086691079935417e+47[0m[90m
      at Context.<anonymous> (tests/test-YRX3.js:100:10)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
[0m
[0m  5) Contract: YRX3
       test for YRX3:
[0m[31m     AssertionError: expected promise to be rejected with 'Error' but it was fulfilled with {}[0m[90m
  
[0m


------------|----------|----------|----------|----------|----------------|
File        |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
------------|----------|----------|----------|----------|----------------|
 contracts/ |    60.75 |       40 |    72.92 |    61.21 |                |
  YRX3.sol  |    60.75 |       40 |    72.92 |    61.21 |... 866,867,868 |
------------|----------|----------|----------|----------|----------------|
All files   |    60.75 |       40 |    72.92 |    61.21 |                |
------------|----------|----------|----------|----------|----------------|

> Istanbul reports written to ./coverage/ and ./coverage.json
> solidity-coverage cleaning up, shutting down ganache server
Error: ‚ùå 5 test(s) failed under coverage.
    at plugin (/app/node_modules/solidity-coverage/plugins/truffle.plugin.js:121:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
Truffle v5.4.6 (core: 5.4.6)
Node v14.17.5
